docs/UPGRADE-19.0.md

```diff
@@ -449,7 +449,7 @@ Where `"<NODE_GROUP_KEY>"` is the key used in the `eks_managed_node_groups` map
 terraform state mv 'module.eks.module.eks_managed_node_group["<NODE_GROUP_KEY>"].aws_iam_role_policy_attachment.this["<POLICY_ARN>"]' 'module.eks.module.eks_managed_node_group["<NODE_GROUP_KEY>"].aws_iam_role_policy_attachment.additional["<POLICY_MAP_KEY>"]'
 ```
 
-### Self-Managed Node Group IAM Role
```

Comment:
This is the correct form https://docs.aws.amazon.com/eks/latest/userguide/worker.html